lib/routes/home/wallet_dashboard.dart

```diff
@@ -52,19 +53,30 @@ class WalletDashboardState extends State<WalletDashboard> {
                       !widget._accountModel.initial
                   ? FlatButton(
                       onPressed: () {
+                        if (_privacyMode) {
+                          setState(() {
+                            _privacyMode = false;
```

Comment:
Don't we want the privacy mode to be persistent?

Reply:
[Done](https://github.com/breez/breezmobile/pull/406/commits/a7847a34fbe3babaef6bcafd639300d76061e719)